modman

```diff
@@ -2,4 +2,4 @@
 ./test app/code/Pronko/LiqPayTest
 ./admin app/code/Pronko/LiqPayAdmin
 ./gateway app/code/Pronko/LiqPayGateway
-./regirect app/code/Pronko/LiqPayRedirect
\ No newline at end of file
+./redirect app/code/Pronko/LiqPayRedirect
```

Comment:
Thanks 👍 It seems like the directory name should also be changed to "redirect".

Reply:
fixed.